### PR TITLE
[#150232524] Gary wants to access Manage Events page

### PR DESCRIPTION
**Pull request by DanielBrierton**

### CODE REVIEW

File: src/dojos/cd-dojo-details.vue [hunk 1]
```diff
@@ -201,6 +201,15 @@
         }
         return facebookData;
       },
+      isCDFAdmin() {
+        return !!(this.user && this.user.roles && this.user.roles.indexOf('cdf-admin') > -1);
+      },
+      isDojoAdmin() {
+        return this.isCDFAdmin || UsersDojosUtil.hasPermission(this.usersDojos, 'dojo-admin');
```
Comment (Wardormeur) [inline src/dojos/cd-dojo-details.vue]: hasPerm could handle the isCDFAdmin ?

File: package.json [hunk 2]
```diff
@@ -58,6 +58,7 @@
     "cp-translations": "^1.0.64",
     "cross-env": "^4.0.0",
     "css-loader": "^0.28.0",
+    "cypress": "^1.0.2",
```
Comment (Wardormeur) [inline package.json]: Is it used?

File: json-server/users-dojos.js [hunk 3]
```diff
@@ -13,10 +13,6 @@ module.exports = {
       ],
       "userPermissions": [
         {
-          "title": "Dojo Admin",
```
Comment (Wardormeur) [inline json-server/users-dojos.js]: Parent1 was dojo-admin ? Dafuk ?

File: package.json (hunk 2 above)
Comment (DanielBrierton) [reply package.json]: Whoops!

File: json-server/users-dojos.js (hunk 3 above)
Comment (DanielBrierton) [reply json-server/users-dojos.js]: Indeed

File: src/dojos/cd-dojo-details.vue (hunk 1 above)
Comment (DanielBrierton) [reply src/dojos/cd-dojo-details.vue]: `hasPerm` is looking at the UsersDojo, not the user itself where the cdf-admin role is defined.

**Pull request merged**